tensorflow/core/grappler/op_types.cc

```diff
@@ -945,6 +950,7 @@ bool NeverForwardsInputs(const NodeDef& node) {
                                 "BatchNormWithGlobalNormalization",
                                 "FusedBatchNorm",
                                 "FusedBatchNormV2",
+                                "FusedBatchNormV3",
```

Comment:
It seems all three versions of FusedBatchNorm can forward their inputs [here](https://github.com/tensorflow/tensorflow/blob/a12d05f82de28dceb62eaf4bb4b8b9daf3a3057b/tensorflow/core/kernels/fused_batch_norm_op.cc#L1185). @rmlarsen, is it a bug that the FusedBatchNorm and FusedBatchNormV2 are currently listed here?

Reply:
@rmlarsen @reedwm Any news on this? I can also move this change to a separate PR so this can be reviewed and merged independently.

Reply:
I think @rmlarsen is out of office. If you remove the FusedBatchNormV3 addition and maybe create a new PR with that, I can approve and merge this PR.

Reply:
@reedwm Thanks for the heads up. I rebased the PR to remove the `FusedBatchNormV3` changes. This should now be ready to review.